packages/revocation-list-2020/src/revocation-list-2020-status.ts

```diff
@@ -68,7 +70,11 @@ export async function checkStatus(credential: CredentialJwtOrJSON) {
   const result = await transmuteCheckStatus({
     credential,
     documentLoader,
-    suite: [new Ed25519Signature2018(), new JsonWebSignature()],
+    suite: [
+      new Ed25519Signature2018(),
+      new JsonWebSignature(),
+      new DataIntegrityProof({ cryptosuite: eddsaRdfc2022CryptoSuite }),
+    ],
```

Comment:
Why does the revocation list need to add the DataIntegrityProof?

Reply:
Yep, you're right, the `eddsaRdfc2022CryptoSuite` shouldn't impact the `revocation-list-2020-status` package. I'll remove it.